[improve][doc] SEO for "About" and "Get Started" (#664)
apache/pulsar-site: docs/getting-started-helm.md

id: getting-started-helm
title: Run a standalone Pulsar cluster in Kubernetes
sidebar_label: "Run Pulsar in Kubernetes"

> description: Want to run Pulsar in Kubernetes? This section guides you through every step of installing and running Apache Pulsar with Helm on Kubernetes quickly.

Reviewer: ```suggestion
description: Get started with Apache Pulsar on your local machine using Kubernetes.
```

---

This section guides you through every step of installing and running Apache Pulsar with Helm on Kubernetes quickly, including the following sections:

> 1. Install the Apache Pulsar on Kubernetes using Helm
> 2. Start and stop Apache Pulsar
> 3. Create topics using `pulsar-admin`
> 4. Produce and consume messages using Pulsar clients
> 5. Monitor Apache Pulsar status with Prometheus and Grafana

Reviewer: ```suggestion

```
Remove this part since the ToC is apparent on the right panel.

:::

> To run Pulsar with Helm on Kubernetes:

Reviewer: ```suggestion
To run Pulsar with Helm on Kubernetes, follow the steps below.
```
